src/components/Footer/sgds-footer.ts

```diff
@@ -1,118 +1,126 @@
 import { html } from "lit";
 import { property } from "lit/decorators.js";
+import { classMap } from "lit/directives/class-map.js";
+import SgdsLink from "../Link/sgds-link";
 import SgdsElement from "../../base/sgds-element";
+import { HasSlotController } from "../../utils/slot";
 import footerStyle from "./footer.css";
 
 /**
  * @summary The footer contains supporting information for your service at the bottom of your website. All .gov.sg digital services shall contain a Global Footer Bar across all pages. The Global Footer Bar should include the name of the digital service, contact information, a privacy statement and the terms of use.
  *
  * @slot title - The slot for title
  * @slot description - The slot for description
- * @slot items - The slot for the list of footer items
- * @slot social-media - The slot for the list of social media with icons
- *
+ * @slot content - The slot for footer content.
+ * @slot items - (@deprecated) Deprecated since 3.5.3 in favour of `content` slot.
+ * Legacy slot for the list of footer items. Use `content` slot instead.
  */
```

Comment:
since we are still keeping sgds-footer-items , how about we keep items slot 

if item slot is slotted with sgds-footer-items we style it like we used to, if its slotted with other things we don't style it like an empty container. Rather than deprecating items and introducing new content slot?